rust-toolchain

```diff
@@ -1 +1 @@
-nightly-2019-07-31
+nightly-2019-08-25
```

Comment:
I think we should only do one thing in one PR. Please pay attention to this rule in the future.

Reply:
Sorry, just find some dependencies cannot  compile because they remove the feature gate.